pkg/core/helper_test.go

```diff
@@ -411,3 +415,41 @@ func addNetworkFee(bc *Blockchain, tx *transaction.Transaction, sender *wallet.A
 	tx.NetworkFee += int64(size) * bc.FeePerByte()
 	return nil
 }
+
+func invokeNativeContractMethod(chain *Blockchain, hash util.Uint160, method string, args ...interface{}) (*state.AppExecResult, error) {
```

Comment:
I guess a lot of code could make use of it.

Reply:
Also, does it work only for native?

Reply:
Changed the name of the method and replaced some common code of `core` tests.